### ISSUE TITLE
FNX-5009 ⁃ [Bug] Site info panel covers the toolbar

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#1410]
## Steps to reproduce
1. Open a website.
1. Tap the padlock in the toolbar.
    1. Expected behavior
The site info panel is displayed above the toolbar.
    1. Actual behavior
The site info panel covers the toolbar.
    1. Device information

* Android device: Nexus 5 (Android 6), Galaxy Tab 3 (Android 8)
* Fenix version: 4/5

**Comment by Amejia481** [comment]
As the mock-ups show, I think in phones should cover the toolbar, but only in tablets it should be above.
![image](https://user-images.githubusercontent.com/773158/55630149-f76d2100-5782-11e9-83d1-2b90ea48b022.png)

**Comment by Amejia481** [comment]
Tablet UI 
![image](https://user-images.githubusercontent.com/773158/55630253-4024da00-5783-11e9-9388-c2e4c8a90911.png)

**Comment by vesta0** [comment]
Thanks @mozilla-mobile/ast 

@lime124 please verify.

**Comment by brampitoyo** [comment]
@lime124 @vesta0 @sv-ohorvath This is an expected behaviour.

Why does the site info panel cover the address bar?

1. The URL is repeated and expanded in the panel. Expanded because the URL can appear without icons on its side.

2. The security icon is also repeated and expanded in the panel. Expanded because we say what the padlock mean. In the future, we also plan to expand this site security info (e.g. show site owner and certificate issuer).

If the site info panel appears above the toolbar, then we will repeat the same piece of information in two different places, and lose a few pixels of vertical screen real estate. This real estate can be used to show more permissions and site controls in the future.

In additioon, horizontal space is a premium on a phone portrait layout. The only way to make the site info panel able to accommodate all those information is by expanding the panel to be as wide as the toolbar.

This is not an issue on a tablet. Its wide screen gives us a space to make the toolbar wide and site info panel narrower (yet still able to contain all the info).

**TL;DR** – Site info panel covering the toolbar is by design.

**Comment by sv-ohorvath** [comment]
Thanks for the info! I'll close the bug.
